### PR TITLE
API: Fix spelling in comment

### PR DESCRIPTION
**Pull request by raffenet**
## What

Spelling fix.

### CODE REVIEW

**Comment by Sergei-Lebedev** [comment]
ok to test

**Review by Sergei-Lebedev — APPROVED** [review]

File: src/ucc/api/ucc.h [hunk 1]
```diff
@@ -507,7 +507,7 @@ typedef enum {
  *
  */
 typedef enum {
-    UCC_NO_SYNC_COLLECTIVES = 0, /*!< Synchornous collectives */
+    UCC_NO_SYNC_COLLECTIVES = 0, /*!< Synchronous collectives */
     UCC_SYNC_COLLECTIVES    = 1  /*!< Non-synchronous collectives */
```
Comment (raffenet) [inline src/ucc/api/ucc.h:511]: are these comments actually on the wrong lines, too?

Comment (manjugv) [reply src/ucc/api/ucc.h:511]: Yes.

**Comment by manjugv** [comment]
@raffenet Can you please add ANL copyright?

**Review by manjugv — APPROVED** [review]

**Pull request merged**